packages/arb-ts/scripts/cleanCompileContracts.ts

```diff
@@ -0,0 +1,27 @@
+import { getPackagePath } from './util'
+// this breaks on browsers, we can instead do a dynamic require to surface the error cleanly?
+import { execSync } from 'child_process'
+
+const cleanCompileContracts = async () => {
+  const arbosPath = getPackagePath('arbos-precompiles')
+  const ethBridgePath = getPackagePath('arb-bridge-eth')
+  const peripheralsPath = getPackagePath('arb-bridge-eth')
+
+  console.log('Clean building arbos')
+  const stdout1 = execSync(`cd ${arbosPath} && yarn clean:build`)
```

Comment:
Maybe simpler to use`yarn workspace arb-precompiles clean:build` in the package.json instead of needing this script?

Reply:
Nevermind, this is so it works when checked out without the monorepo